Dockerfile

```diff
@@ -44,12 +43,16 @@ COPY templates /var/www/templates
 COPY config /var/www/config
 COPY src /var/www/src
 COPY data /var/www/data
+COPY public /var/www/public
+
+# Copy assets from docker assets stage in a not mapped folder to be maintained
+RUN mkdir -p /assets && mkdir -p /assets/css
+COPY --from=assets /assets/css/*.css /assets/css/
 
 # Reset "local"/development config files
 RUN rm -f /var/www/config/development.config.php && \
-  rm /var/www/config/autoload/*.local.php && \
-  mv /var/www/config/autoload/local.php.dist /var/www/config/autoload/local.php
+    rm -f /var/www/config/autoload/*.local.php && \
+    mv /var/www/config/autoload/local.php.dist /var/www/config/autoload/local.php
```

Comment:
What prompted the indentation changes? Were you seeing errors? If so, can you provide:

- Docker and docker-compose version
- The error message(s) you observed

I don't see any myself when building images, so I'm curious about shi.

Reply:
There was no error related to the indentation, it was just to order the code.
The only problem was the lack of the parameter `-f` in the command `rm`.

OS: Mac OS
Docker version 18.09.2
docker-compose version 1.23.2